src/commands/moderator/LoggingSync.ts

```diff
@@ -0,0 +1,104 @@
+import {BaseCommand, ICommandContext, ICommandInfo} from "../BaseCommand";
+import {MongoManager} from "../../managers/MongoManager";
+import {MessageUtilities} from "../../utilities/MessageUtilities";
+import {Logger} from "../../utilities/Logger";
+
+const LOGGER: Logger = new Logger(__filename, false);
+export class LoggingSync extends BaseCommand {
+    private _isRunning: boolean = false;
+
+    public constructor() {
+        const cmi: ICommandInfo = {
+            cmdCode: "LOGGING_SYNC_COMMAND",
+            formalCommandName: "Logging Sync",
+            botCommandName: "loggingsync",
+            description: "Updates list of users who have logged items for this guild.",
```

Comment:
What is this command doing, exactly?

Reply:
Updating each guild's usersWithLogs to capture all the data up to this point. Future entries are automatically populated by the logging manager.